### ISSUE TITLE
yaml-tokens example doesn't produce expected results

### ISSUE THREAD

**Issue by jbarreiros** [amzn/style-dictionary#510]
Pulled down `3.0.0-rc.1` to test it out and it has some great stuff. The transitive feature is :hot_pepper: 

Testing out the "yaml-tokens" example, and when I run `npm run build`, it completes, but the final result (`build/variables.css`) looks like:
```css
/**
 * Do not edit directly
 * Generated on Thu, 17 Dec 2020 17:52:16 GMT
 */

:root {

}
```

Interestingly, if I put the following at the top of `sd.config.js`, it shows that the `yaml` package is interpreting the yaml file correctly.
```js
const yaml = require('yaml');
const fs = require('fs');

const yamlraw = fs.readFileSync('./tokens/color/core.yaml', {encoding: 'UTF-8'});
const result = yaml.parse(yamlraw);
console.log(result);
```
```js
{ color:
   { base:
      { white: [Object],
        red: [Object],
        green: [Object],
        success: [Object] } } }
```

**Comment by dbanksdesign** [comment]
There was an issue that no parsers were being run because we changed the method signature of combineJSON.. but I posted a PR for that fix. Thanks for bringing this up!

**Comment by chazzmoney** [comment]
This is fixed now in the 3.0 branch and will be part of 3.0.0-rc.2
